docs/src/main/sphinx/connector/hive.rst

```diff
@@ -388,6 +388,11 @@ Property Name                                      Description
 ``hive.query-partition-filter-required``           Set to ``true`` to force a query to use a partition filter.   ``false``
                                                    You can use the ``query_partition_filter_required`` catalog
                                                    session property for temporary, catalog specific use.
+
+``hive.statistics_enabled``                        Disable :doc:`/optimizer/statistics`, and therefore           ``true``
+                                                   :doc:`/optimizer/cost-based-optimizations`. You can use the 
+                                                   ``statistics_enabled`` catalog session property for 
+                                                   temporary, catalog specific use.         
```

Comment:
Both catalog and session properties are "catalog specific". I prefer the existing wording in some other configs like:

```rst
  * - ``join-pushdown.enabled``
    - Enable :ref:`join pushdown <join-pushdown>`. Equivalent :doc:`catalog
      session property </sql/set-session>` is ``join_pushdown_enabled``.  Enabling
      this may negatively impact performance for some queries.
    - ``false``
```

Reply:
Seems to be a pattern in this file though.